### ISSUE TITLE
Breadcrumb

### ISSUE THREAD

**Issue by eddywashere** [reactstrap/reactstrap#76]
http://v4-alpha.getbootstrap.com/components/breadcrumb/

- [x] Breadcrumb
- [x] BreadcrumbItem (w/ active)

**Comment by TheSharpieOne** [comment]
An interesting thing is the "without list markup" which bootstrap v4 documents. The way it is achieved with the PR i put up is with the use of overriding the tag via the `tag` prop. This seems cumbersome, but since most of the time dev user would override the the `a` tag with something like`{Link}`, it doesn't seems to bad.
Perhaps in the future, a prop on the `Breadcrumb` component would help it's child `BreadcrumbItem`s.
IDK, something like 
```
<Breadcrumb item={Link}>
  <BreadcrumbItem to={...}>Home</BreadcrumbItem>
  <BreadcrumbItem to={...}>Library</BreadcrumbItem>
  <BreadcrumbItem tag="span" active>Data</BreadcrumbItem>
</Breadcrumb>
```
where it would use `item` (or it could be called `itemTag`) to tell the `BreadcrumbItem`s what tag to use by default (and if item is present, default it's own tag to `nav`) and thus produce this:
```
<nav class="breadcrumb">
  <a class="breadbcrumb-item" href="...">Home</a>
  <a class="breadbcrumb-item" href="...">Library</a>
  <span class="breadbcrumb-item active">Data</span>
</nav>
```

Let me know what you think, this can easily be a non breaking enhancement after the PR which is currently there.

**Comment by eddywashere** [comment]
@TheSharpieOne that would be a good thing to introduce to root level components that have item or inner components that depend on the root class or tag. I think that would be a great use of context.

**Comment by eddywashere** [comment]
closed via #85. Let's move discussion of this sort of change to a new issue
